modules/gitops-release-notes-1-15-1.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-15.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-15-1_{context}"]
+= Release notes for {gitops-title} 1.15.1
+
+{gitops-title} 1.15.1 is now available on {OCP} 4.14, 4.15, 4.16, and 4.17.
+
+[id="errata-updates-1-15.1_{context}"]
+== Errata updates
+
+[id="RHBA-XXXX:NNNN-gitops-1-15-1-security-update-advisory_{context}"]
```

Comment:
Remember to sub out `XXXX:NNNN` when appropriate, of course.

Reply:
Fixed.